Packs/InfobloxBloxOne/Integrations/InfobloxBloxOneThreatDefense/InfobloxBloxOneThreatDefense.py

```diff
@@ -0,0 +1,190 @@
+    params = task.get('params', {})
+    return {
+        'task_id': task.get('task_id'),
+        'type': params.get('type'),
+        'target': params.get('target'),
+        'source': params.get('source')
+    }
+
+
+def dossier_source_list_command(client: BloxOneTDClient) -> CommandResults:
+    sources = client.dossier_source_list()
+    return CommandResults(
+        outputs_prefix='BloxOneTD',
+        outputs={'DossierSource': sources},
+    )
+
+
+def validate_and_format_lookalike_domain_list_args(args: Dict) -> Dict:
+    match len(list(filter(bool, [args.get('filter'), args.get('target_domain'), args.get('detected_at')]))):
+        case 0:
+            raise ValueError("You must spesify one of the following arguments 'target_domain', 'detected_at' or 'filter'.")
+        case 1:
+            pass
+        case _:
+            raise ValueError("You can spesify one of the following arguments 'target_domain', 'detected_at' or 'filter' not more.")
+
+    if args.get('detected_at'):
+        try:
+            args['detected_at'] = dateparser.parse(args['detected_at']).isoformat()
+        except AttributeError:
+            raise DemistoException(f"could not parse {args['detected_at']} as a time value.")
+    return args
+
+
+def lookalike_domain_list_command(client: BloxOneTDClient, args: Dict) -> CommandResults:
+    args = validate_and_format_lookalike_domain_list_args(args)
+
+    data = client.lookalike_domain_list(
+        user_filter=args.get('filter'),
+        target_domain=args.get('target_domain'),
+        detected_at=args.get('detected_at'),
+        limit=int(args.get('limit', 50)),
+        offset=int(args.get('offset', 0))
+    )
+    return CommandResults(
+        outputs_prefix='BloxOneTD.LookalikeDomain',
+        outputs=data,
+    )
+
+
+def dossier_lookup_get_command_results(data: Dict) -> CommandResults:
+    headers_map = {'task_id': 'Task Id', 'type': 'Type', 'target': 'Target', 'source': 'Source'}
+    outputs = list(map(dossier_lookup_task_output, data.get('results', [])))
+    readable_output = tableToMarkdown(name='Lookalike Domain List', t=outputs, headers=[], headerTransform=lambda x: headers_map[x])
+    return CommandResults(
+        outputs_prefix='BloxOneTD.DossierLookup',
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=data
+    )
+
+
+def dossier_lookup_get_schedule_polling_result(args: Dict) -> CommandResults:
+    next_run_in_seconds = int(args.get('interval_in_seconds', 10))
+    timeout_in_seconds = int(args.get('timeout', 600))
+    args['timeout'] = timeout_in_seconds - next_run_in_seconds
+    scheduled_command = ScheduledCommand(
+        command='bloxone-td-dossier-lookup-get',
+        next_run_in_seconds=next_run_in_seconds,
+        timeout_in_seconds=timeout_in_seconds,
+        args=args
+    )
+
+    return CommandResults(
+        readable_output='Job is still running, it may take a little while...',
```

Comment:
If you remove this part, there won't be any print after the first execution. Please consider avoiding further prints after the first one.